cdap-ui/app/cdap/components/ResourceCenterEntity/ResourceCenterEntity.scss

```diff
@@ -69,4 +58,46 @@ $height-of-image: 75px;
       cursor: pointer;
     }
   }
+  .entity-image {
+    width: $height-of-image;
+    height: $height-of-image;
+    display: inline-block;
+    vertical-align: top;
+    font-size: $height-of-image;
+    &.empty {
+      background: gray;
+    }
+
+    .icon-svg {
+      vertical-align: inherit;
+    }
+  }
+
+  &.resourcecenter-entity-card-pipeline {
```

Comment:
If this is a separate component we should have the styles in its own scss file. Would be easier to track and debug if its inside the component.